flycheck-rust.el

```diff
@@ -145,12 +145,25 @@ description of the conventional Cargo project layout."
                       (file-equal-p dir (file-name-directory target-path))))
                   ;; build a list of (target . dir) candidates
                   (-table-flat
-                   'cons targets
+                   'cons (-flatten-n 1 targets)
                    (flycheck-rust-dirs-list file-name
                                             (file-name-directory manifest)))))
             ;; If all else fails, just pick the first target
-            (car targets))))
-      (let-alist target (cons (flycheck-rust-normalize-target-kind .kind) .name)))))
+            (car (car targets)))))
+      (let-alist target
+        ;; If target is 'custom-build', we search other target
```

Comment:
Please change `we search other target` to `we pick another target from the same package (see GH-62)`.